build/yaml/crd/vpc/crd.nsx.vmware.com_ipaddressallocations.yaml

```diff
@@ -63,8 +64,8 @@ spec:
           status:
             description: IPAddressAllocationStatus defines the observed state of IPAddressAllocation.
             properties:
-              CIDR:
```

Comment:
We also support PrivateTGW type for IPAddressBlockVisibility

Reply:
ok